README.md

```diff
@@ -537,8 +537,4 @@ It is important to briefly explain how the communication between DKG ceremony In
 4. Operators verify each of the messages from other Operators participating in the ceremony and verifies Initiator's signature of the combined message.
 5. During the DKG protocol execution, the BLS auth scheme is used - G2 for its signature space and G1 for its public keys
 
-More details in the [Flow Description](#flow-description) section.
-
----
-
-More info about how things are designed/work under the hood can be found [here](./design.md)
```

Comment:
or just link to the architecture pic?

Reply:
I moved architecture to `design.md`. Link was broken probably because of merge. Im going to merge it with this PR:

https://github.com/bloxapp/ssv-dkg/pull/20/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R537-R543

Reply:
I see. I can revert this change then.